src/plugins/expressions/common/execution/execution.test.ts

```diff
@@ -426,6 +426,68 @@ describe('Execution', () => {
         });
       });
     });
+
+    test('ignores partial results by default', () => {
```

Comment:
So, I guess this means that we won't have to do anything on the Lens side to take advantage of this change.